frontend/src/plugins/api-token.server.ts

```diff
@@ -1,11 +1,13 @@
+import { defineNuxtPlugin, useRuntimeConfig } from "#imports"
```

Comment:
Should this module move to server/plugins?

Reply:
It probably should move to `server/plugins`, or to a separate module for the API interaction, but I think it's best to defer this to a separate follow-up issue. It is working as is, so I'd prefer not to touch it yet :)